links.json

```diff
@@ -55,21 +55,21 @@
                     "type": "string"
                 },
                 "targetSchema": {
-                    "$ref": "http://json-schema.org/draft-08/hyper-schema#"
+                    "$recursiveRef": "http://json-schema.org/draft-08/hyper-schema"
```

Comment:
Since there is no `"$recursiveRoot"` in links.json when used as an entry point, each time we reference a document with a `"$recursiveRoot"`, the root is set just for that reference and any further references it makes.  Then when we move "out" of this `"$recursiveReference"` and on to the next one for some other LDO property, the process repeats.

This still works fine when we start from links.json, `"$recursiveRef"` into hyper-schema, and then reference _back_ to links.json again.  The sequence is:

* Enter links.json (no `"$recursiveRoot"`)
* `"$recursiveRef"` to hyper-schema (use the URI reference directly as there's no root yet)
    * Enter hyper-schema.json, see `"$recursiveRoot"` for the first time, set the recursion target
    * `"$ref"` to schema.json (extending the core/validation meta-schema)
        * Enter schema.json `"$recursiveRoot"`, but ignore it because the recursion has already been set
        * Any `"$recursiveRef"`s encountered, such as for the applicator keywords, go to the recursion target, which is hyper-schema.json
    * Back out in hyper-schema.json, `"$ref"` to links (this is never recursive)
        * Re-enter links.json, the recursive root is still set from hyper-schema.json
        * Encounter a `"$recursiveRef"` to hyper-schema, but now with the recursion target set use that (in this case, it happens to point to the same place)
        * This continues as we walk over the instance and bounce back and forth between links.json, hyper-schema.json, and schema.json